lsp-completion.el

```diff
@@ -761,8 +761,8 @@ The CLEANUP-FN will be called to cleanup."
                                (setq-local lsp-inhibit-lsp-hooks nil))))
     (cond
      (lsp-completion-mode
-      (setq-local completion-at-point-functions nil)
-      (add-hook 'completion-at-point-functions #'lsp-completion-at-point nil t)
+      (make-local-variable 'completion-at-point-functions)
+      (add-to-list 'completion-at-point-functions #'lsp-completion-at-point)
```

Comment:
The `completion-at-point-functions` is a special hook, can we use `add-hook` here?